src/rust/src/x509/certificate.rs

```diff
@@ -412,6 +412,17 @@ fn parse_display_text(
         DisplayText::IA5String(o) => Ok(pyo3::types::PyString::new(py, o.as_str()).to_object(py)),
         DisplayText::Utf8String(o) => Ok(pyo3::types::PyString::new(py, o.as_str()).to_object(py)),
         DisplayText::VisibleString(o) => {
+            if !UnvalidatedVisibleString::verify(o.as_str()) {
```

Comment:
Rather than do this, can we do `if asn1::VisibleString::new(o.as_str()).is_none()`?